src/pfe/file-watcher/server/test/scripts/exec.sh

```diff
@@ -72,8 +92,13 @@ function setup {
 }
 
 function run {
-    docker exec -it $CODEWIND_CONTAINER_ID bash -c "cd /file-watcher/server; JUNIT_REPORT_PATH=/test_output.xml npm run $TEST_SUITE:test:xml" | tee $TEST_LOG
-    docker cp $CODEWIND_CONTAINER_ID:/test_output.xml $TEST_OUTPUT
+    if [ $TEST_TYPE == "local" ]; then
+        docker exec -it $CODEWIND_CONTAINER_ID bash -c "cd /file-watcher/server; JUNIT_REPORT_PATH=/test_output.xml npm run $TEST_SUITE:test:xml" | tee $TEST_LOG
+        docker cp $CODEWIND_CONTAINER_ID:/test_output.xml $TEST_OUTPUT
+    elif [ $TEST_TYPE == "kube" ]; then
+        kubectl exec -it $CODEWIND_POD_ID -- bash -c "cd /file-watcher/server; JUNIT_REPORT_PATH=/test_output.xml npm run $TEST_SUITE:test:xml" | tee $TEST_LOG
```

Comment:
We can also make `"cd /file-watcher/server; JUNIT_REPORT_PATH=/test_output.xml npm run $TEST_SUITE:test:xml"` a variable and call it in both docker and kube exec

Reply:
tried it, it did not work